mimic/rest/auth_api.py

```diff
@@ -224,6 +228,27 @@ def get_username(self, request, tenant_id):
         session = self.core.sessions.session_for_tenant_id(tenant_id)
         return json.dumps(dict(user=dict(id=session.username)))
 
+    @app.route('/v2.0/users', methods=['GET'])
+    def get_users_details(self, request):
+        """
+        Returns response with  detailed account information about each user
+        including email, name, user ID, account configuration and status
+        information.
+        """
+        username = request.args.get("name")
+        session = self.core.sessions.session_for_username_password(
+            username[0], "test")
+        return json.dumps(dict(user={
+            "RAX-AUTH:domainId": session.tenant_id,
+            "id": session.user_id,
+            "enabled": True,
+            "username": session.username,
+            "email": "thisisrandom@email.com",
+            "RAX-AUTH:defaultRegion": "ORD",
+            "created": seconds_to_timestamp(time.time()),
+            "updated": seconds_to_timestamp(time.time())
+        }))
```

Comment:
I understand, but as Mimic has no preset set of users as such we decided that until a use case requires otherwise, a GET call will fetch an existing session or will create session in the absence of an existing session.

My usecase in an ele test in the branch https://github.com/racker/ele/pull/2913 in `test-customer-client.js` test in which the GET on the `/users` occurs after have gotten the user from the customer api.